package.json

```diff
@@ -50,6 +51,7 @@
     "debug": "^4.1.1",
     "express": "^4.16.4",
     "mongoose": "^5.4.9",
+    "nodemon": "^1.18.9",
```

Comment:
I'm removing the nodemon from our repo since it introduced a lot of useless packages.
I suggest everyone to install nodemon globally using `npm install -g nodemon`.